apps/etterlatte-behandling/src/main/kotlin/Contekst.kt

```diff
@@ -74,6 +74,13 @@ class Saksbehandler(
         )
     }
 
+    fun harRolleFortrolig(): Boolean {
+        return identifiedBy.getJwtToken(AZURE_ISSUER).jwtTokenClaims.containsClaim(
+            "groups",
+            saksbehandlerGroupIdsByKey["AZUREAD_FORTROLIG_GROUPID"] ?: ""
```

Comment:
Er det sånn at claimet er garantert å ikke ha tom streng/gruppe i seg? Hvis ikke vil man få tilgang til alle roller i Doffen, eller? 🤔

Reply:
Det skal være garantert i behandlingsappen, dette var satt på tidligere for å unngå at noe ikke fungerte som jeg ikke helt husker hva var. Det er kanskje best om vi fjerner det slik at ikke tilfeldigvis vi får ok på et claim som er lik `""`?

Reply:
Det virker umiddelbart som en ryddigere approach å kaste feil hvis appen ikke har rett oppstartsparametre enn å defaulte til noe som potensielt er feil.

Reply:
Ser forøvrig ut som akkurat denne metoden ikke blir brukt lengre.

Reply:
Fikset i https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/1219/commits/d560dab02268ed3d7d5b34d4a8ec5a69e017311a